frontend/src/components/file-explorer/FileExplorer.tsx

```diff
@@ -243,6 +275,17 @@ function FileExplorer({ error, isOpen, onToggle }: FileExplorerProps) {
               <p className="text-neutral-300 text-sm">{error}</p>
             </div>
           )}
+          {isOpen && (
+            <button
+              type="button"
+              onClick={handleVSCodeClick}
+              className="mt-auto mb-2 w-full h-10 bg-[#0066B8] hover:bg-[#005BA4] text-white rounded flex items-center justify-center gap-2 transition-colors"
```

Comment:
Let's make this our normal purple, instead of VScode-blue. Or maybe even something subtler like the gray

<img width="378" alt="Screenshot 2024-11-09 at 9 58 29 AM" src="https://github.com/user-attachments/assets/021cbd3e-bad2-4bc0-8675-74f89a6f0b80">